go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-openapi/spec v0.20.3
```

Comment:
this addition looks like a bad rebase

Reply:
Showed up when I ran hack/update-vendor.sh

Reply:
make sure you're rebased on latest master and `make clean` first?

Reply:
```
get fetch upstream
git rebase upstream/master
make clean
./hack/pin-dependency.sh sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.15
./hack/update-vendor.sh
```
Still getting it. (go 1.16.4)